### PR TITLE
Add 64 bit ABIs, bump SDK_VER

### PR DESCRIPTION
**Pull request by pazos**
Requires https://github.com/koreader/koreader-base/pull/1230
Requires https://github.com/koreader/android-luajit-launcher/pull/253

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/virdevenv/54)

### CODE REVIEW

**Review by Frenzie — CHANGES_REQUESTED** [review]
Please change https://github.com/koreader/virdevenv/blob/b963d2381e0da01ff4115886d804e358129b695f/docker/ubuntu/koandroid/Makefile#L1 to 0.4.0 or 1.0.0 or something. ;-)

File: docker/ubuntu/koandroid/build_android_tc.sh [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 echo "Pruning NDK to reduce image size..."
 NDK=android-ndk-r15c
-# only keep NDK 14
-rm -rf ${NDK}/platforms/android-{9,12,13,15,16,17,18,19,21,22,23,24,26}
-# only keep arm and x86
-rm -rf ${NDK}/toolchains/{aarch64-linux-android-4.9,mips64el-linux-android-4.9,mipsel-linux-android-4.9,x86_64-4.9}
+# only keep platform-14 (32bit ABIs) and platform-21 (64bit ABIs) 
+rm -rf ${NDK}/platforms/android-{9,12,13,15,16,17,18,19,22,23,24,26}
```
Comment (Frenzie) [inline docker/ubuntu/koandroid/build_android_tc.sh:6]: Nothing higher than 26 in that newer one, like 27 or 28?

Comment (Frenzie) [reply docker/ubuntu/koandroid/build_android_tc.sh:6]: Never mind, still the same NDK here. Minor mind hiccup wrt that issue from earlier about NDK 18.

Comment (pazos) [reply docker/ubuntu/koandroid/build_android_tc.sh:6]: Yup. `platform` should match minSdkVersion for each ABI. Actually we're using `platform-14` everywhere, but https://github.com/koreader/android-luajit-launcher/pull/196 showcases how we need to use `platform-21` on 64bit abis

**Comment by Frenzie** [comment]
If you think it's ready, please merge https://github.com/koreader/koreader-base/pull/1230 and update this line to finish this PR:

https://github.com/koreader/virdevenv/blob/b963d2381e0da01ff4115886d804e358129b695f/docker/ubuntu/koandroid/fetch_android_tc.sh#L5

Then I can build and push the new image. :-)

**Comment by pazos** [comment]
> Then I can build and push the new image. :-)

@Frenzie: done. Not sure if the new image will work until https://github.com/koreader/android-luajit-launcher/pull/253 is merged. It should work as the SDK won't find SDK28 and will try to download it, but cannot be sure how it will behave on a docker container. Same¿?

**Comment by Frenzie** [comment]
@pazos The image (pushing as we speak; should be uploaded in a minute or two) won't be used until https://gitlab.com/koreader/nightly-builds/-/blob/24768a7a5d5f9091f14288d092496394eacc3009/.gitlab-ci.yml#L131 (and the equivalent line in the CircleCI config in base) is updated. Right now it's just preparations so that the switch will only take seconds.

**Comment by Frenzie** [comment]
But yes, it'll behave exactly the same, perhaps even too much so. The only thing you'll notice if you pay attention is that the build will take some 5-10 minutes longer than it should.

**Pull request merged**